platforms/web/test-e2e/tests/home_test.ts

```diff
@@ -39,9 +39,11 @@ Scenario('Header button navigates to playlist screen', async ({ I }) => {
   I.see('The Daily Dweebs');
 });
 
-Scenario('I can slide within the featured shelf', async ({ I }) => {
+// @todo: this test requires the testconfig to have `layoutType: 'hero'` on the first shelf
+// eslint-disable-next-line codeceptjs/no-skipped-tests
+Scenario.skip('I can slide within the hero shelf', async ({ I }) => {
```

Comment:
Do we expect it to start working when we add a custom param to the first shelf of `gnnuzabk` config?

Reply:
Yes, I have tested that, but then the featured test will break. 
Should we change the tests and config to the following layoutTypes?
- Shelf 0: `hero`
- Shelf 1: undefined
- Shelf 2: `featured`
I suggest shelf 2 for featured, because shelf 1 is already an exception with the different aspect ratio's.

Alternatively we could change this later.

Reply:
Considering the time, I suggest we postpone this test? @AntonLantukh

Reply:
Perhaps we can make a separate config for this in the next sprint?

Reply:
It also works for me